context/input.go

```diff
@@ -301,6 +301,12 @@ func (input *BeegoInput) SetParam(key, val string) {
 	input.pnames = append(input.pnames, key)
 }
 
+// ResetParams clears any of the input's Params
```

Comment:
Could you add more comments why add this function? That would be great to call back when refactor these function in future.